src/plugins/album-actions/index.ts

```diff
@@ -62,7 +62,7 @@ export default createPlugin<
       //Finds the playlist
       const playlist =
         document.querySelector('ytmusic-playlist-shelf-renderer') ??
-        document.querySelector('ytmusic-shelf-renderer')!;
+        [...document.querySelectorAll('ytmusic-shelf-renderer')].at(-1)!;
```

Comment:
Have you tried CSS pseudo selectors, e.g. `:last`, and does it not work?
If it does not work, then I suggest the below patch to avoid allocating an array.

```suggestion
        Array.prototype.at.call(document.querySelectorAll('ytmusic-shelf-renderer'), -1)!;
```